.github/CODEOWNERS

```diff
@@ -1255,6 +1261,12 @@ x-pack/test_serverless/**/test_suites/observability/ai_assistant @elastic/obs-ai
 ### END Observability Plugins
 
 # Presentation
+/x-pack/test/accessibility/apps/group3/maps.ts  @elastic/kibana-presentation
+/x-pack/test/accessibility/apps/group3/graph.ts  @elastic/kibana-presentation
+/x-pack/test/accessibility/apps/group3/enterprise_search.ts  @elastic/kibana-presentation
```

Comment:
This should be @elastic/search-design maybe?

Reply:
https://github.com/elastic/kibana/pull/195964/commits/124fb81695c1edbd53ed984ec92cc540386ce58d